src/modules/Dashboard/features/Applet/Popups/DuplicatePopups/DuplicatePopups.tsx

```diff
@@ -132,6 +125,27 @@ export const DuplicatePopups = ({ onCloseCallback }: { onCloseCallback?: () => v
     duplicatePopupsClose();
   };
 
+  const handleDuplicateSuccess = () => {
+    setPasswordModalVisible(false);
+
+    onCloseCallback?.();
+    duplicatePopupsClose();
+
+    Mixpanel.track('Applet Created Successfully');
```

Comment:
This is being called twice. It is previously called [here](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1493/files#diff-63de0e2eaed09f83b62776df97142dd861fbdd856d16b9de58a7c741422f88beL87)

Might make sense to keep this one and remove the other one.

Reply:
@mbanting  thank you for the catch! @sabrichu could you please add the tests to cover the new functionality?

And I'd like to remind the whole team, we need to make it a rule as a part of PR checklist (yes, we have it, but sometime we forget about it :). 
Team please don't hesitate to remind about test coverage that we must cover any new functionality well with tests.

This is our defense and protection , keeps the functionality safe from possible errors and maintains the quality of the product when adding new functionality.

Reply:
Great catch @mbanting! And thanks for the reminder @alex-i-e, will do 🙌

Reply:
Added tests for `useSaveAndPublishSetup`